oscarlab/graphene
# Testing rebased version of #734

This is to test #734 after squash and rebase. Not meant for merging.



## Affected components


## Description of the changes 

See #734.

## How to test this PR? 

See #734.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/933)
